storage/tables/data-types/index.md

```diff
@@ -993,29 +993,111 @@ The `TIMESTAMP` base type represents a date value with a time portion.
 </tr>
 </table>
 
-### Native datatypes
+## Native datatypes
 
-As mentioned above, Keboola stores data in storage as text by default. Native datatypes feature breaks this paradigm. If this feature is enabled, data in storage are stored in columns which have datatypes as described in Keboola metadata. 
+As mentioned above, Keboola stores data in Storage as text by default. Native datatypes feature breaks this paradigm. If this feature is enabled, data in Storage are stored in columns with datatypes based on Keboola metadata. 

```

Comment:
Tady na začátek bych přidal, že to jak to funguje závisí na backendu a hlavně link na backends a dál se věnujeme hlavně snowflake backendu

Reply:
Žádnou stránku "backends" nemáme :)

Reply:
https://help.keboola.com/storage/#storage-data